tests/unit/linters/test_classic_linter.py

```diff
@@ -90,6 +91,12 @@ def test_classic_linter(mocker, new_dir, confinement, stage_libc, text):
                 filename="elf.bin",
                 text="ELF rpath should be set to '/snap/core22/current/lib/x86_64-linux-gnu'.",
             ),
+            LinterIssue(
```

Comment:
I thought the issue was that the linter *shouldn't* warn about libs without an interpreter section? Or is this checking something else?

Reply:
ah yes that makes sense, thanks